stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messagepreview/MessagePreviewView.kt

```diff
@@ -4,15 +4,33 @@ import android.content.Context
 import android.util.AttributeSet
 import android.view.LayoutInflater
 import android.widget.FrameLayout
-import com.getstream.sdk.chat.utils.DateFormatter
 import io.getstream.chat.android.client.models.Message
 import io.getstream.chat.android.client.models.name
 import io.getstream.chat.android.ui.databinding.StreamUiMessagePreviewItemBinding
+import io.getstream.chat.android.ui.utils.DateFormatter
+import io.getstream.chat.android.ui.utils.formatMessageDate
 
 public class MessagePreviewView : FrameLayout {
 
     private val binding = StreamUiMessagePreviewItemBinding.inflate(LayoutInflater.from(context), this, true)
 
+    private var _dateFormatter: DateFormatter? = null
+
+    /**
+     * The formatter used to display the time/date for the message.
+     * If not set explicitly, a default implementation will be used.
```

Comment:
This is to allow usage of this Custom View outside of the adapters we ship, which will always set this property.